crates/ra_ide_api/src/completion/completion_item.rs

```diff
@@ -70,6 +73,9 @@ impl fmt::Debug for CompletionItem {
         if let Some(documentation) = self.documentation() {
             s.field("documentation", &documentation);
         }
+        if let Some(deprecated) = self.deprecated {
+            s.field("deprecated", &deprecated);
```

Comment:
let's not print anything unless the item is deprecated. That way, we won't be adding a new line to every completion item

Reply:
Makes sense, I'll do that.